appium/tests/screenobjects/attachment.screen.ts

```diff
@@ -2,7 +2,9 @@ import BaseScreen from './base.screen';
 import ElementHelper from "../helpers/ElementHelper";
 
 const SELECTORS = {
-  BACK_BTN: '~arrow left c'
+  BACK_BTN: '~arrow left c',
+  SAVE_BTN: '-ios class chain:**/XCUIElementTypeButton[`label == "Save"`]',
+  CANCEL_BTN: '~Cancel',
 };
```

Comment:
I'll file issue to add accessibility ids to these three, `aid-back-icon`, `aid-save-attachment-to-device` and `aid-cancel-saving-attachment`